lint/lint_private.go

```diff
@@ -42,6 +43,38 @@ const (
 	attrVeryOpinionated
 )
 
+type parameters map[string]interface{}
+
+func (p parameters) Int(key string, defaultValue int) int {
```

Comment:
Hmm... The disadvantage of such API is that default values are not very apparent and will be spread all across the checker code. But that can work for now! Better than not having config at all.
We can adapt later.